literature-cleaned.bib

```diff
@@ -2526,6 +2563,19 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{PRS+:WWW16,
+  author = {Pezoa, Felipe and Reutter, Juan L. and Suarez, Fernando and Ugarte, Mart\'{\i}n and Vrgo\v{c}, Domagoj},
+  title = {{Foundations of JSON Schema}},
+  year = 2016,
+  publisher = {International World Wide Web Conferences Steering Committee},
+  booktitle = {Proceedings of the 25th International Conference on World Wide Web},

```

Comment:
As we usually reuse booktitle values with variables, they are not year-specific. So I would suggest to drop the `25th` for consistency, and optionally to extract into variable.

Reply:
For further consistency, please abbreviate `Proceedings` etc as in the values in MYabrv:
`@String{ICSE = "Proc.\ Int'l Conf.\ on Software Engineering (ICSE)"}`